pkg/controllers/webhook/controller.go

```diff
@@ -509,6 +509,11 @@ func (c *controller) buildVerifyMutatingWebhookConfiguration(caBundle []byte) (*
 						"app.kubernetes.io/name": kyvernov1.ValueKyvernoApp,
 					},
 				},
+				NamespaceSelector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app.kubernetes.io/name": kyvernov1.ValueKyvernoApp,
```

Comment:
`kyvernov1.ValueKyvernoApp` is a constant, this will not work is deployed in another namespace.

You can use `config.KyvernoNamespace()`.

Reply:
Thanks for that insight.